observation_portal/requestgroups/serializers.py

```diff
@@ -231,10 +231,14 @@ def validate_instrument_configs(self, value):
         return value
 
     def validate_instrument_type(self, value):
-        if value and value not in configdb.get_active_instrument_types({}):
+        is_staff = False
```

Comment:
Could do `is_staff = self.context.get('request') or False`

Reply:
just having a request context doesn't make it staff, the requests user must be staff. I think it might get ugly if I try to make that a single line...